### PR TITLE
[INLONG-4602][Manager] Remove the ClusterBean class

### LINKED ISSUE #4602: [Improve][Manager] Remove the ClusterBean class
### Description

Remove the ClusterBean class, because it was not needed anymore.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ciscozhou**
### Prepare a Pull Request

- Fixes #4602 

### Motivation

Remove the ClusterBean class, because it should be replaced by cluster APIs.

### Modifications

Remove the ClusterBean class.

### Verifying this change

- [x] This change is a trivial rework/code cleanup without any test coverage.

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: inlong-manager/manager-web/src/main/resources/application-prod.properties [hunk 1]
```diff
@@ -54,19 +54,10 @@ cluster.tube.master=127.0.0.1:8000,127.0.0.1:8010
 # Tube cluster ID
 cluster.tube.clusterId=1
```
Comment (healchow) [inline inlong-manager/manager-web/src/main/resources/application-prod.properties:55]: Please check if this parameter is used. If not, it is recommended to delete it.

File: inlong-manager/manager-web/src/main/resources/application-prod.properties [hunk 2]
```diff
@@ -54,19 +54,10 @@ cluster.tube.master=127.0.0.1:8000,127.0.0.1:8010
...
 
-# Push configuration to the path on ZooKeeper
-cluster.zk.url=127.0.0.1:2181
-cluster.zk.root=inlong_hive
-
-# Application name in Sort
-sort.appName=inlong_app
-
 # Pulsar admin URL
 pulsar.adminUrl=http://127.0.0.1:8080,127.0.0.2:8080,127.0.0.3:8080
```
Comment (healchow) [inline inlong-manager/manager-web/src/main/resources/application-prod.properties:58]: Please check if those Pulsar parameters are used. If not, it is recommended to delete it.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/ConsumptionServiceImpl.java [hunk 3]
```diff
@@ -418,6 +402,12 @@ private void fullConsumptionInfo(ConsumptionInfo info) {
                 streamTopics.forEach(stream -> topicSet.remove(stream.getMqResource()));
                 Preconditions.checkEmpty(topicSet, "topic [" + topicSet + "] not belong to inlong group " + groupId);
             }
+            InlongGroupEntity inlongGroupEntity = groupMapper.selectByGroupId(groupId);
+            if (null != inlongGroupEntity) {
+                info.setTopic(String.format(InlongGroupSettings.PULSAR_TOPIC_FORMAT, "public",
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/ConsumptionServiceImpl.java:407]: The default tenant is `public`, but before the default value, we should query from the cluster first.

Please refer to the `InlongClusterService#getOne` to get more information.

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**